apps/docs/src/pages/index.tsx

```diff
@@ -1,296 +1,66 @@
-              <h2 className="text-3xl font-bold tracking-tight text-gray-900 sm:text-4xl">
-                {title}
-              </h2>
-              <p className="mt-6 text-lg leading-8 text-gray-600">
-                {description}
-              </p>
-              <dl className="mt-10 max-w-xl space-y-8 text-base leading-7 text-gray-600 lg:max-w-none">
-                {features.map((feature) => (
-                  <div key={feature.title} className="relative pl-9">
-                    <dt className="inline font-semibold text-gray-900">
-                      <FontAwesomeIcon
-                        aria-hidden="true"
-                        icon={feature.icon}
-                        className="text-uzh-red-100 absolute left-1 top-1 h-5 w-5"
-                      />
-                      {feature.title}
-                    </dt>{' '}
-                    <dd className="inline">{feature.text}</dd>
-                  </div>
-                ))}
-              </dl>
-            </div>
-          </div>
-          <img
-            src={imgSrc}
-            alt="Product screenshot"
-            className="w-full max-w-none rounded-xl shadow-xl ring-1 ring-gray-400/10 sm:w-[57rem] md:-ml-4 md:w-[48rem] lg:-ml-0"
-            // width={2432}
-            // height={1442}
-          />
-        </div>
-      </div>
-    </div>
-  )
-}
-
-function UseCaseOverview() {
-  return (
-    <div className="space-y-4">
-      <H2 className={{ root: 'text-2xl' }}>Use Cases</H2>
-      <div className="grid grid-cols-1 gap-4 rounded md:grid-cols-3">
-        {Object.entries(USE_CASES).map(([href, item]) => (
-          <div
-            key={item.title}
-            className="flex flex-col justify-between gap-4 p-4 shadow"
-          >
-            <div className="flex-1">
-              <H3>{item.title}</H3>
-              <Prose className={{ root: 'prose-sm' }}>{item.abstract}</Prose>
-            </div>
-            <div className="flex-none">
-              <a
-                className="inline-flex items-center gap-2"
-                href={`/use_cases/${href}`}
-              >
-                <FontAwesomeIcon icon={faArrowRight} />
-                <div>More Details</div>
-              </a>
-            </div>
-          </div>
-        ))}
-      </div>
-    </div>
-  )
-}
+import { CTA } from '../components/landing/CTA'
+import { FeatureFocusSection } from '../components/landing/FeatureFocusSection'
+import FeatureSection from '../components/landing/FeatureSection'
+import { TitleImage } from '../components/landing/TitleImage'
+import { UseCaseOverview } from '../components/landing/UseCaseOverview'
 
 function Home() {
   return (
     <Layout>
       <TitleImage />
       <FeatureSection
-        title="Synchronous Interaction"
-        description="Interact your students during class and drive engagement with
-          your materials."
-        imgSrc="/img_v3/sync_interaction.png"
+        title="Gamified Learning"
```

Comment:
Maybe we could make three different blocks like this underneath each other? One for gamification elements, one for synchronous activities and one for asynchronous activities? With somewhat reduced spacing between the text blocks, we could probably even fit it in the same space

Reply:
It might be worth checking, whether alternating the size of the image makes it cooler for three successive blocks (like e.g. done on https://www.gbl.uzh.ch/) or whether it just becomes chaotic